draft-ietf-rtcweb-jsep.xml

```diff
@@ -1140,7 +1140,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTrack" anchor="sec.addTrack">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTrack method adds a MediaStreamTrack to the PeerConnection.
```

Comment:
Need to reference the MediaStreams argumetn here, i.e. regarding grouping/lip-sync, as we will refer to the MediaStream grouping in that section.